Umbraco-StarterKit/Widgets/Grid/index.md

```diff
@@ -4,51 +4,68 @@ versionFrom: 8.0.0
 
 # The Grid
 
-The Grid Widget lets you set up framework of widgets, you will have the ability to make it look the way u want, you can do so by setting up the general look of the grid, and choose what widgets you wnat your grid to contain.
+The Grid Widget lets you set up a framework of widgets, you will have the ability to make it look the way u want. You can do so by setting up the general look of the grid, and choose what widgets you want your grid to contain.
 
-(explain about rows and coulmns and more indepth about what is a grid)
+## Columns
+
+A Column is the boxes wherein you create your content when working in a grid. Most of the grid widgets require you to set a column width. This is because it sets the width for how big the box will be and by that determine how many consecutive Columns can be in a row.
```

Comment:
Instead of "most of the grid widgets", I think it would be best to say "all" - even though it currently isn't a requirement on all, it really should be.

```suggestion
Columns are the boxes wherein you create your content when working in a grid. Most of the grid widgets require you to set a column width. This is because it sets the width for how big the box will be and by that determine how many consecutive columns can be in a row.
```